app/models/story.rb

```diff
@@ -183,7 +184,8 @@ def date_for_boolean(value)
 
   # raising exceptions here to prevent sending publish messages
   #  when not actually a change to be published
-  def publish!(apublished_at = DateTime.now)
+  def publish!(apublished_at = nil)
```

Comment:
yeah, good one - could get rid of this arg, especially now with `released_at` as the mechanism to set published_at